### PR TITLE
[4800][ADD] stock_lot_purchase_attribute

### PR DESCRIPTION
**Pull request by yostashiro**
[4800](https://www.quartile.co/web#id=4800&cids=3&model=project.task&view_type=form)

### CODE REVIEW

**Comment by AungKoKoLin1997** [comment]
@yostashiro I didn't use limit and `_trigger()` for each limit records because the total running time will be the same with all records once.

File: stock_lot_purchase_attribute/readme/DESCRIPTION.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This module adds fields to stock.lot to keep some information from the related purchase
+order.
```
Comment (yostashiro) [inline stock_lot_purchase_attribute/readme/DESCRIPTION.md]:
```suggestion
This module adds fields to stock.lot to keep some information from the related purchase
order. Corresponding fields are also added to relevant transactions, such as stock.move.line
and stock.quant.
```

**Review by yostashiro — COMMENTED** [review]
@AungKoKoLin1997 Thanks for the update.  Can you please help add simple tests.

@kanda999 Can you please add a translation and update the test database.

File: stock_lot_purchase_attribute/tests/test_stock_lot_purchase_attribute.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        )
+
+    def test_stock_lot_purchase_attribute(self):
+        # Confirm the purchase order
+        self.purchase_order.button_confirm()
+        self.assertEqual(self.purchase_order.state, "purchase", "PO not confirmed")
+
+        picking = self.purchase_order.picking_ids
+        self.assertTrue(picking, "No incoming picking found.")
+
```
Comment (yostashiro) [inline stock_lot_purchase_attribute/tests/test_stock_lot_purchase_attribute.py]: * No need to comment the obvious.
* No need to test something unrelated to the module function, unless it means something in the test scenario.
* Avoid inserting blank lines in a method.

```suggestion
        self.purchase_order.button_confirm()
        picking = self.purchase_order.picking_ids
```

**Review by AungKoKoLin1997 — APPROVED** [review]

**Review by kanda999 — APPROVED** [review]
Functional and Code review.

**Pull request merged**